h2o-py/tests/testdir_algos/gbm/pyunit_PUBDEV-6754_gbm_ntree_actual.py

```diff
@@ -0,0 +1,30 @@
+from builtins import range
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.gbm import H2OGradientBoostingEstimator
+
+def gbm_ntree_actual():
+  prostate = h2o.import_file(path=pyunit_utils.locate("smalldata/logreg/prostate.csv"))
+  prostate[1] = prostate[1].asfactor()
+  prostate.summary()
+  ntrees_original = 1000
+
+  prostate_gbm = H2OGradientBoostingEstimator(nfolds=5,ntrees=ntrees_original, distribution="bernoulli", stopping_metric="MSE", stopping_tolerance=0.01, stopping_rounds=5)
+  prostate_gbm.train(x=list(range(2,9)), y=1, training_frame=prostate)
+  
+  print("\n")
+  print("number of trees set by user before building the model is:")
+  print(ntrees_original)
+  print("number of trees built with early-stopping is:")
+  print(prostate_gbm.ntree_actual())
+  
+  assert prostate_gbm.ntree_actual() == prostate_gbm._model_json['output']['model_summary']['number_of_trees'][0]
```

Comment:
I would also suggest checking that the model actually stopped early. That is actual number of trees < 1000.

Reply:
done